isis/src/base/apps/gaussstretch/gaussstretch.cpp

```diff
@@ -0,0 +1,57 @@
+#include "ProcessByLine.h"
+#include "Statistics.h"
+#include "GaussianStretch.h"
+#include "gaussstretch.h"
+
+using namespace std;
+using namespace Isis;
+
+namespace Isis {
+  void gauss(Buffer &in, Buffer &out);
+
+  vector<GaussianStretch *> stretch;
```

Comment:
This definition can go inside the `gaussstretch(Cube *icube, UserInterface &ui)` as it will be captured by the lambda.